build-release-tools/lib/common.py

```diff
@@ -126,3 +128,28 @@ def str2bool(string):
     else:
         return False
 
+def retry(ExceptionToCheck, tries=3, delay=5, backoff=2):
```

Comment:
maybe 'retry' and 'deco_retry' could be some other names which are more straight forward